lms/templates/registration/password_reset_complete.html

```diff
@@ -6,49 +6,30 @@
 {% endblock %}
 
 {% block bodyextra %}
-  <script type="text/javascript">
-    $(function() {
-
-      // adding js class for styling with accessibility in mind
-      $('body').addClass('js');
-
-      // new window/tab opening
-      $('a[rel="external"], a[class="new-vp"]')
-      .click( function() {
-      window.open( $(this).attr('href') );
-      return false;
-      });
-
-      // form field label styling on focus
-      $("form :input").focus(function() {
-        $("label[for='" + this.id + "']").parent().addClass("is-focused");
-      }).blur(function() {
-        $("label").parent().removeClass("is-focused");
-      });
-    });
-  </script>
+    <script type="text/javascript">
+        $(function() {
+            'use strict';
+            // adding js class for styling with accessibility in mind
+            $('body').addClass('js');
+        });
+    </script>
 {% endblock %}
 
 {% block bodyclass %}view-passwordreset{% endblock %}
 
 {% block body %}
-  <section class="introduction">
-    <header>
-      <h1 class="title">
-        <span class="title-super">
-          {% trans "Your Password Reset is Complete" %}
-        </span>
-      </h1>
-    </header>
-  </section>
-
-  <section class="passwordreset container">
-    {% block content %}
-    <section role="main" class="content">
-      {% blocktrans with link_start='<a href="/login">' link_end='</a>' %}
-      Your password has been set.  You may go ahead and {{ link_start }}log in{{ link_end }} now.
-      {% endblocktrans %}
+<div id="password-reset-complete-container" class="login-register">
+    <section id="password-reset-complete-anchor" class="form-type">
+        <div id="password-reset-complete" class="form-wrapper" aria-hidden="true">
```

Comment:
@waheedahmed this perpetuates a bug tracked by https://openedx.atlassian.net/browse/ECOM-3938 and shouldn't be merged until this is resolved. As @cptvitamin notes, this kind of bug is covered in by our [a11y dev guide](https://edx.readthedocs.org/projects/edx-developer-guide/en/latest/conventions/accessibility.html#some-cautions-for-using-wai-aria) (third bullet). @cptvitamin would you mind signing off on the fix once @waheedahmed is able to get to it?